hash/spectrum_cass.xml

```diff
@@ -97592,7 +98094,7 @@ Lacks game introduction, bad dump or alternate version?
 	<software name="brunilda">
 		<description>Los Amores de Brunilda (v1.4, English)</description>
 		<year>2014</year>
-		<publisher>Retroworks</publisher>
+		<publisher>RetroWorks</publisher>
```

Comment:
It’s “Retroworks” – see the title and footer of their web site: http://www.retroworks.es/php/game_en.php?id=9

Please back out all of these.

Reply:
I don't totally agree with this one.
On that same [page](http://www.retroworks.es/php/game_en.php?id=9) (a little bit above) you can see:
![image](https://github.com/mamedev/mame/assets/26376777/ee78ad64-e670-4b86-a4bc-2357e3cd466b)

On the footer of the "[About Us](http://www.retroworks.es/nosotros_en.php)" page, it's written "RetroWorks".
On [facebook](https://www.facebook.com/people/RetroWorks/100063785823298/) it's the same.
At last, on [X](https://twitter.com/retroworks_es?lang=en) page, also "RetroWorks".

So, if they haven't changed recently the way the name is written, I think it should be "RetroWorks".
Or maybe, both ways are valid...

Reply:
They do indeed seem to use both interchangeably.

RetroWorks here: http://www.retroworks.es/php/game_en.php?id=16
Both ways here: http://www.retroworks.es/php/game.php?id=16